docs/concepts/fundamentals/render_api.md

```diff
@@ -49,42 +46,159 @@ rendered = Table.render(
 
 The Render API includes:
 
-- [`self.id`](../render_api/#component-id) - The unique ID for the current render call
+- [`self.args`](../render_api/#args) - The positional arguments for the current render call
+- [`self.kwargs`](../render_api/#kwargs) - The keyword arguments for the current render call
+- [`self.slots`](../render_api/#slots) - The slots for the current render call
 - [`self.input`](../render_api/#component-inputs) - All the component inputs
+- [`self.id`](../render_api/#component-id) - The unique ID for the current render call
 - [`self.request`](../render_api/#request-object-and-context-processors) - The request object (if available)
 - [`self.context_processors_data`](../render_api/#request-object-and-context-processors) - Data from Django's context processors (if request is available)
 - [`self.inject()`](../render_api/#provide-inject) - Inject data into the component
 
-## Component ID
```

Comment:
The section on component ID wasn't deleted, only moved to [line 231](https://github.com/django-components/django-components/pull/1205/files#diff-9fbdf12fc67b61f0e13b4b521ee3375a107cdf4a87b9b5403a453540ec06bdd5R231).